sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
+        if "vti" in str(device) and "ip_vti0" not in str(device):
```

Comment:
There could be more than one vti, so we should not specify vti0.  I think `"ip_" not in str(device):` should suffice.  Otherwise this is good.